polars/polars-lazy/polars-plan/src/logical_plan/format.rs

```diff
@@ -174,7 +174,14 @@ impl LogicalPlan {
             }
             Aggregate {
                 input, keys, aggs, ..
-            } => write!(f, "Aggregate\n\t{:?} BY {:?} FROM {:?}", aggs, keys, input),
+            } => {
+                writeln!(
+                    f,
+                    "{:indent$}Aggregate\n\t{:?} BY {:?} FROM",
```

Comment:
So the first indentation is correct. But then there is a new line followed by a tab, e.g. `\n\t` that should also be indented by `{:indent}` instead of the tab.

```
"{:indent$}Aggregate\n{:indent}{:?} BY {:?} FROM",
```

Reply:
Ahh yes sorry I completely missed that one. Think I've addressed this now.